contracts/eth/nearprover/utils/upgrade_bridge_address.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config();
+
+const { ethers } = require('hardhat');
+const { assert, expect } = require('chai');
+
+const BRIDGE_ADDRESS_SLOT = 2;
+
+async function upgradeProversBridgeAddressTo(provider, proverAddress, newBridgeAddress) {
+    const nearProverFactory = await ethers.getContractFactory('NearProver');
+    const nearProver = nearProverFactory.attach(proverAddress);
+
+    console.log(`Got prover at address: ${proverAddress}`);
+
+    const initialBridgeAddress = await nearProver.bridge();
+    console.log(`Initial bridge address: ${initialBridgeAddress}`);
+    console.log(`Trying to upgrade bridge address to: ${newBridgeAddress}`);
+
+    const adminWallet = new ethers.Wallet(process.env.ETH_PRIVATE_KEY, provider);
+
+    assert.equal(
+        adminWallet.address,
+        await nearProver.admin(),
+        "The used account is not an admin of NearProver"
+    );
+
+    // Mask matches only on the latest 20 bytes (to store the address)
+    const mask = ethers.BigNumber.from("0x000000000000000000000000ffffffffffffffffffffffffffffffffffffffff");
+    const response = await nearProver
+        .connect(adminWallet)
+        .adminSstoreWithMask(BRIDGE_ADDRESS_SLOT, newBridgeAddress, mask);
+    await response.wait(10).then(function(receipt) {
+        console.log(`Transaction mined: `);
+        console.log(receipt);
+    });
```

Comment:
```suggestion
    let receipt = await response.wait(10);
    console.log('Transaction mined: ');
    console.log(receipt);
```

Reply:
Remember that 10 is the number of blocks so this is over 2 minutes (~140s)

Reply:
It is, though if the receipt is available earlier, it will return the receipt immediately.

Reply:
Interesting, I thought it was waiting for 10 blocks on top of the block where the receipt was included.